share/spice/sublime_packages/footer.handlebars

```diff
@@ -1,4 +1,10 @@
 <div class="tile__update__info one-line">
-	<i class="ddgsi ddgsi-arrow-down"></i>
-	{{unique_installs}} &middot; Updated {{last_modified}}
+        <span class="opt--left">
```

Comment:
Ah I understand now that the `opt--*` only take effect on small screens, so we want to use pull--* for it to affect all screens.